scripts/projects.py

```diff
@@ -23,7 +24,9 @@
 
 
 class Project:
-    def __init__(self, project: str, fuzzdir: str, dumpdir: str):
+    def __init__(
+        self, project: str, fuzzdir: str, dumpdir: str, proj_language: str = "c"
```

Comment:
we can just use "lang" for project language since it has no other meaning in this context